docs/getting-started-guides/logging-elasticsearch.md

```diff
@@ -1,21 +1,26 @@
 ---
 ---
 
-On the Google Compute Engine (GCE) platform the default cluster level logging support targets
-[Google Cloud Logging](https://cloud.google.com/logging/docs/) as described at the [Logging](/docs/getting-started-guides/logging) getting
-started page. Here we describe how to set up a cluster to ingest logs into Elasticsearch and view them using Kibana as an
-alternative to Google Cloud Logging.
-
-To use Elasticsearch and Kibana for cluster logging you should set the following environment variable as shown below:
+On the Google Compute Engine (GCE) platform, the default logging support targets
```

Comment:
Yes, but the instructions about how to use an environment variable to change that won't work on GKE, so I'm hesitant to bring that up here. I've added a little note indicating as such.